plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryTableHandle.java

```diff
@@ -130,40 +125,36 @@ public boolean equals(Object o)
     @Override
     public int hashCode()
     {
-        return Objects.hash(projectId, schemaName, tableName, type, constraint, projectedColumns, limit);
+        // NOTE remoteTableName is not hashed here because two handles differing in only remoteTableName will create ambiguity
+        // TOOD: Add tests for this
```

Comment:
The comment here is redundant over equals's comment.
hashCode is just a servant of equals